src/core/connection.c

```diff
@@ -1441,13 +1441,16 @@ QuicConnTryClose(
     }
 
     if (!ClosedRemotely) {
-
         if ((Flags & QUIC_CLOSE_APPLICATION) &&
-            Connection->Crypto.TlsState.WriteKey < QUIC_PACKET_KEY_1_RTT) {
```

Comment:
It's possible for the server to close the connection before we have 1-RTT keys I believe (though very rare). I feel like we should keep the old logic for server too.